drivers/clock_control/clock_control_wch_rcc.c

```diff
@@ -34,6 +35,15 @@
 #define WCH_RCC_SRC_IS_HSI 1
 #endif
 
+#if defined(CONFIG_SOC_CH32V307)
+/* TODO: Entry 13 is 6.5x (fractional multiple currently unsupported without
+ * changes to RCC config datatype)
+ */
+static const uint8_t pllmul_lut[] = {18, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 0, 15, 16};
```

Comment:
nit: call this wch_rcc_pllmul_lut, even though it's static.

Reply:
Okay - will do.
I'm planning on adding some ethernet subsystem drivers soon, will lump that change in with that. 👍